### DISCUSSION TITLE
missing "smooth" param in useScroll

### DISCUSSION THREAD

**Discussion by aquibbaig** [motiondivision/motion#2917] [Q&A] [answered]
Seems like "smooth" parameter has been deprecated from useScroll (version: 12.0.0-alpha.2) 

```
// 'smooth' does not exist in type 'UseScrollOptions'
const { scrollY } = useScroll({ smooth: 0.4 });
```

Is it explicit now, or has it been replaced with -- It'd be great if someone can point me to the commit/issue/docs.

**Comment by mattgperry** [comment, marked as answer]
You can smooth out a scroll value with useSpring. This was removed from the types but was actually never implemented.

**Reply by aquibbaig** [reply]
thanks! :-)
